### PR TITLE
chore(store): update eu stores

### PR DESCRIPTION
**Pull request by fnk93**
### Description
Fix euroFormat for several EU stores.
Add Xbox Series X, PS5 disc/digital to Euronics (DE).
Add PS5 disc bundle to Alternate.
Change test item on several EU Amazon sites to a product available.

### CODE REVIEW

File: src/store/model/euronics-de.ts [hunk 1]
```diff
@@ -8,20 +8,52 @@ export const EuronicsDE: Store = {
...
+			]
 		}
 	},
 	links: [
+		{
+			brand: 'test:brand',
+			model: 'test:model',
+			series: 'test:series',
+			url:
+				'https://www.euronics.de/telefon-und-navigation/festnetz/schnurlose-telefone/kx-tg6721gb-schnurlostelefon-mit-anrufbeantworter-schwarz-4051168442801'
```
Comment (jef) [inline src/store/model/euronics-de.ts:28]: It seems that this content is in fact in `euroFormat`

`31,16 €`

**Review by jef — CHANGES_REQUESTED** [review]
Could you explain a little more on the euroformat changes? Thanks!

File: src/store/model/euronics-de.ts (hunk 1 above)
Comment (fnk93) [reply src/store/model/euronics-de.ts:28]: Just for my understanding, as I think I may have misunderstood the use of euroFormat.
Is it to toggle between "." for true and "," for false as separator between the thousands?
If yes, then give me some minutes, I'll fix the mess I did here.

**Comment by joerg-knitter** [comment]
Why does the Alternate link refer to the MEGA bundle of the PS5 console, not the "standard" edition without bundled games?

**Comment by fnk93** [comment]
@joerg-knitter I only added the mega bundle, the standard disc url has been in already before and is still in there: https://www.alternate.de/product/1651220

File: src/store/model/euronics-de.ts (hunk 1 above)
Comment (jef) [reply src/store/model/euronics-de.ts:28]: Yep! That's exactly it. Thanks for fixing and also updating old ones 😄

**Review by jef — APPROVED** [review]
~I think the containers in `euronics-de.ts` are wrong? But that's not your doing. I'll have to go in and test.~ It's fine actually. Sorry about that!

Thanks!

**Comment by fnk93** [comment]
Already have been panicking and looking for the html source to find my mistake.

**Pull request merged**